.github/workflows/ci.yaml

```diff
@@ -0,0 +1,109 @@
+name: CI
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+#    branches:
+#      - master
+#      - lsof # just to be able to run all these builds on our own fork
+jobs:
+  with-docker-root:
+    #if: ${{ false }}  # disable for now
+    name: Inside Container centos-zulu-${{matrix.java}} as root
+    runs-on: ubuntu-latest
+    container:
+      image: centos:latest
+    strategy:
+      matrix:
+        java: [ 8, 11 ]
+      fail-fast: false
+      max-parallel: 4
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - uses: actions/setup-java@v2
+        with:
+          distribution: zulu
+          java-version: ${{matrix.java}}
+      - name: Test with Maven
+        run: ./mvnw clean verify -V -B -D"maven.artifact.threads=64" -D"org.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn" -D"disable-toolchain" -D"java.version=${{matrix.java}}" -D"inside-container=true"
```

Comment:
Quotes are likely unnecessary in all mvn command lines here.

Reply:
@mathieucarbou is correct that the quoting would be necessary for Windows `bat` -- but the quoting that's normally used is to quote the entire Java argument -- e.g. `"-Darg=value"` instead of `-D"foo=bar"`.  (It would also be proper to simply _escape_ the equals sign using a caret [`^`] as in `-Dfoo^=bar`.)  An unprotected equals sign in a Windows command line is taken to be a token separator.

Reply:
I think I cannot do all that because the same invocation (as we see now) has not be compatible with **every** OS...

Also, it seems to me that `-D"maven.artifact.threads=64"` is an equivalent to `-D "maven.artifact.threads=64"` (usually a space delimiting the parameter name and its value being an association of "k=v" here).

Reply:
It's equivalent only because the shell strips the quotes -- somewhat indiscriminately.  If you were constructing the command line in code, as a Java `Process` for example, you would not include quotes where you've placed them -- the argument is `-Darg=value`.  Quoting the full argument -- `"-Darg=value"` -- is more consistent and is valid in any shell context to which I have access.  (And a space is _not_ permitted between the `-D` and the property name/value -- it's interpreted by `java` and a separate argument unrelated to the `-D`.)

Additionally, to what is `inside-container=true` relevant?  I see the referent for `disable-toolchain` in the pom but nothing obviously related to `inside-container`.

Reply:
This system prop is required to disable the toolchain plugin because it is not needed when running in a controlled environment like in GA build where only one jvm is installed. There is also no toolchain file to query in any image.

Reply:
I will update the parameter lines tomorrow

Reply:
- removed `inside-container` (was not used, probably my first attempt to do that)
- renamed to "skipToolchain"
- updated quotes (put them only where needed)